students/jemielity_kamil/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+path = 'Path\\to\\directory'
+
+for foldername, subfolders, filenames in os.walk(path):
```

Comment:
For unused variables there is convention to use '_' but is is not stated in PEP8
for foldername, _, filenames in os.walk(path):